### DISCUSSION TITLE
Block Syncing issues in erigon Polygon Node

### DISCUSSION THREAD

**Discussion by HarishankarVj12** [erigontech/erigon#9349] [Q&A]
`INFO[01-30|12:36:48.195] Req/resp stats                           req=0 reqMin=0 reqMax=0 skel=0 skelMin=0 skelMax=0 resp=0 respMin=0 respMax=0 dups=0
INFO[01-30|12:37:08.195] [2/15 Headers] No block headers to write in this log period block number=43999999
INFO[01-30|12:37:08.195] Req/resp stats                           req=0 reqMin=0 reqMax=0 skel=0 skelMin=0 skelMax=0 resp=0 respMin=0 respMax=0 dups=0
INFO[01-30|12:37:26.039] [txpool] stat                            pending=0 baseFee=0 queued=0 alloc=2.8GB sys=4.9GB
INFO[01-30|12:37:28.194] [2/15 Headers] No block headers to write in this log period block number=43999999
INFO[01-30|12:37:28.194] Req/resp stats                           req=0 reqMin=0 reqMax=0 skel=0 skelMin=0 skelMax=0 resp=0 respMin=0 respMax=0 dups=0`

I've initiated the deployment of my Erigon node on the server with the specified configuration, including an 8-core CPU at 4.5 GHz, 64GB of DDR5 RAM at 4800, dual 1TB NVMe disks, and a 3Gbps uplink NIC. However, during the syncing process, I encountered a recurring issue where the output logs displayed the same information as mentioned in our previous conversation. Please advise on how to resolve this issue and guide me on checking the current synchronization progress of the node in terms of the synced blocks.

Server Configuration:
CPU: 8 Cores @ 4.5 GHz
RAM: RAM DDR5 64GB 4800
Disk: 2x NVMe 1TB
NIC : 3Gbps Uplink

**Comment by AskAlexSharov** [comment]
pls use bsc-erigon's discord server for BSC support
